src/assignments/yunfeng/advent_of_code/2018/day_2/code.clj

```diff
@@ -0,0 +1,58 @@
+(ns advent-of-code.2018.day-2.code
+  (:require [clojure.java.io :as io]))
+
+(defn frequency-accumulator
```

Comment:
Code written in one long line like this
is impossible to read.

Try something like the following, with minor
modifications to enhance readability:

```
(defn frequency-accumulator
  "Return a new map of format like {:2 x :3 y} and help counter-for-23 to accumulate the result"
  [frequency-for-23 new-frequency]
  (let [frequency-vec (vals new-frequency)
        freq-2 (if (some #{2} frequency-vec) 1 0)
        freq-3 (if (some #{3} frequency-vec) 1 0)]
    (merge-with +
                frequency-for-23
                {:2 freq-2
                 :3 freq-3})))
```